externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/events/handler/SparkJsonLoggingEventHandlerSuite.scala

```diff
@@ -46,6 +46,16 @@ class SparkJsonLoggingEventHandlerSuite extends WithSparkSQLEngine with HiveJDBC
 
   override protected def jdbcUrl: String = getJdbcUrl
 
+  override def afterAll(): Unit = {
+    super.afterAll()
+    val fileSystem: FileSystem = FileSystem.get(new Configuration())
```

Comment:
Utils.createTempDir() will deleteOnExist？ is is not working？

Reply:
I see, I will close it.